akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -806,12 +768,37 @@ private[akka] class Shard(
     }
   }
 
+  // FIXME in what scenario do we get this to the shard?
```

Comment:
I haven't quite figured this out. When remember entities, there is a `StartEntity` -> `StartEntityAck` exchange with the `RememberEntitiesStarter`, and the `StartEntity` can come from the outside, but I don't find anywhere that the shard itself sent the `StartEntity` and should get the ack.

Reply:
the starter forwards the ack to the shard

Reply:
Yep, I figured that out and messed with it in #29164